src/time.ts

```diff
@@ -122,35 +139,20 @@ export class CronTime {
 			this.realDate && this.source instanceof DateTime
 				? this.source
 				: DateTime.local();
-		if (this.zone) {
-			date = date.setZone(this.zone);
+		if (this.timeZone) {
+			date = date.setZone(this.timeZone);
 		}
 
-		if (this.utcOffset != null) {
-			const offsetHours = parseInt(
-				// @ts-expect-error old undocumented behavior going to be removed in V3
-				this.utcOffset >= 60 || this.utcOffset <= -60
-					? // @ts-expect-error old undocumented behavior going to be removed in V3
-					  this.utcOffset / 60
-					: this.utcOffset
-			);
+		if (this.utcOffset !== undefined) {
+			const sign = this.utcOffset < 0 ? '-' : '+';
 
-			const offsetMins =
-				// @ts-expect-error old undocumented behavior going to be removed in V3
-				this.utcOffset >= 60 || this.utcOffset <= -60
-					? // @ts-expect-error old undocumented behavior going to be removed in V3
-					  Math.abs(this.utcOffset - offsetHours * 60)
-					: 0;
-			const offsetMinsStr = offsetMins >= 10 ? offsetMins : `0${offsetMins}`;
+			const offsetHours = Math.trunc(this.utcOffset / 60);
+			const offsetHoursStr = Math.abs(offsetHours);
```

Comment:
a nitpick but this technically isn't a string as far as I can tell

Reply:
I got you :wink: